ci-operator/templates/cluster-launch-e2e-azure.yaml

```diff
@@ -177,6 +179,8 @@ objects:
         mountPath: /tmp/shared
       - name: cluster-secrets-azure
         mountPath: /etc/azure/credentials
+      - name: artifacts
+        mountPath: /tmp/artifacts
```

Comment:
for e2e test we will need writable /tmp/openshift . wondering is it worth replacing all those empty dir with one /tmp and share it?

Reply:
Why /tmp/openshift? I am using the same path used elsewhere?

Reply:
Origin tests default to those. Now I point them to:
```
export LOG_DIR=/tmp/shared/logs
export VOLUME_DIR=/tmp/shared/volume
export ARTIFACT_DIR=/tmp/shared/ARTIFACT
export FAKE_HOME_DIR=/tmp/shared/home
```
so potentially we will have emptyDirs: `shared-data`, `artifacts`, `origin-e2e` ? Maybe 3 is not so bad... (we dont use now `shared-data`)

Reply:
Where did you read these from? [This](https://github.com/openshift/release/blob/e0d96e76939fa812510dbb2388c95f73bd868b13/ci-operator/templates/cluster-launch-e2e.yaml#L83) is the origin e2e template mount.

Reply:
this is `origin` `TEST_ONLY=1 test/extended/core.sh` stuff im trying to get in :)

Reply:
lets merge this and worry later